packages/react/src/components/Masthead/MastheadTopNav.js

```diff
@@ -64,7 +64,7 @@ const MastheadTopNav = ({ navigation, ...topNavProps }) => {
     } else {
       return (
         <HeaderMenuItem
-          aria-selected={selected ? 'true' : 'false'}
+          selected={selected ? 'true' : 'false'}
```

Comment:
looks like it should be `data-selected`?

Reply:
Yeah 😩